src/constants/dashboard-constants.ts

```diff
@@ -0,0 +1,13 @@
+import { COLORS } from "../theme"
+
+export const CHALLENGE_IMAGE_SOURCE: { [key: string]: number } = {
+    '1': require('../assets/icons/dActive.png'),
```

Comment:
This is already present in notification-constants bcoz it has already been used in the notifications screen